Cargo.toml

```diff
@@ -10,13 +10,13 @@ readme = "README.md"
 documentation = "https://docs.rs/st7735-lcd"
 
 [dependencies]
-embedded-hal = "0.2"
-nb = "0.1"
+embedded-hal = "1.0.0-rc.1"
```

Comment:
I think we should wait until this is stable.

Reply:
it is 1.0 now